src/Main.java

```diff
@@ -2,5 +2,31 @@
 public class Main {
 //2. Maak binnen de main-klasse een main-methode aan.
 public static void main(String[] args) {
+    //Create Pokemon objects
+    Pikachu pikachu = new Pikachu("Pikachu", 10, 50, "Thunderbolt" );
+    Charmander charmander = new Charmander("Charmander", 8, 45, "Flamethrower");
+    Balbasaur balbasaur = new Balbasaur("Balbasaur", 7, 40, "Vine Whip");
+    Squirtle squirtle = new Squirtle("Squirtle", 9, 42, "Water Gun");
+
+    //Use methods
+    pikachu.attack();
```

Comment:
Je print statments staan nu in de methode. Probeer ook eens om het pas in de main() te printen.